net/obfsproxy/files/obfsproxy.conf

```diff
@@ -0,0 +1,57 @@
+# To use Obfsproxy with Tor, follow the instructions at:
+#
+# https://www.torproject.org/projects/obfsproxy-instructions.html.en
+#
+# instead of setting up a separate instance of Obfsproxy.
+
+config obfsproxy 'obfsproxy'
+	# Set to 1 to enable this instance
+	option enabled 0
+
+	# One of: managed, dummy, b64, obfs2, obfs3, scramblesuit
+	option transport 'scramblesuit'
+
+	# Shared secret / password
+	# For obfs2 (as the shared secret parameter) and scramblesuit only
+	option password 'EXAMPLEPASSWORDNOTREAL'
+
+	# One of: server, ext_server, client, socks
+	option mode 'socks'
+
+	# Destination address
+	# Required for all modes except 'socks'
+	#option dest_host '0.0.0.0'
+	#option dest_port '80'
+
+	# Extended ORPort authentication cookie file location
+	# Required for 'ext_server' mode
+	#option ext_cookie_file ''
+
+	# Listener address
+	option listen_host '127.0.0.1'
+	option listen_port '8080'
+
+	# Set to log to a file instead of syslog
+	#option log_file '/var/log/obfsproxy.log'
+
+	# Minimum logging severity
+	# One of: error, warning, info, debug
+	#option log_min_severity 'info'
+
+	# Set to 1 to disable logging
+	#option no_log 0
+
+	# Set to 1 to disable safe (scrubbed address) logging
+	#option no_safe_logging 0
+
+	# Run as a different user
+	#option user 'obfsproxy'
```

Comment:
Daemons should run with, or drop to retain only minimal privileges.  Well, it's true that many packages do not specifically enforce on this, it's just still the best practice.

If user 'obfsproxy' is hinted, then we'd better create such a user to make the default consistent.